modules/codegen/src/main/scala/com/twilio/guardrail/CLI.scala

```diff
@@ -96,7 +96,7 @@ trait CLICommon {
 
     val fallback = List.empty[Path]
     import CLICommon.unsafePrintHelp
-    val /*(logger,*/ paths /*)*/ = result
```

Comment:
If you wouldn't mind leaving these comments, I'm in the middle of a refactor to try and put them back without tanking performance

Reply:
Resolving this